### ISSUE TITLE
kijk.nl: detection

### ISSUE THREAD

**Issue by thijshofkes14** [uBlockOrigin/uAssets#24948]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://www.kijk.nl/programmas/red-mijn-vakantie/RfPeD2rtSUi`

### Category

detection

### Description

    It is detected

### Other extensions used

none

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.59.0
Chromium: 126
filterset (summary):
 network: 153397
 cosmetic: 47946
 scriptlet: 22101
 html: 0
listset (total-discarded, last-updated):
 default:
  user-filters: 0-0, never
  easylist: 85209-567, 53m Δ
  easyprivacy: 52976-628, 53m Δ
  plowe-0: 3547-0, now
  ublock-badware: 10361-6, 53m Δ
  ublock-filters: 39661-284, 53m Δ
  ublock-privacy: 1065-2, 53m Δ
  ublock-quick-fixes: 168-0, 53m Δ
  ublock-unbreak: 2496-19, 53m Δ
  urlhaus-1: 29620-0, 53m
filterset (user): [empty]
trustedset:
 added: [array of 9 redacted]
userSettings:
 suspendUntilListsAreLoaded: true
hiddenSettings: [none]
supportStats:
 allReadyAfter: 1111 ms (selfie)
 maxAssetCacheWait: 99 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 9
 network:
  kijk.nl: 1
  2cnt.net: 1
  cxense.com: 1
  demdex.net: 1
  googletagmanager.com: 1
  newrelic.com: 1
  scorecardresearch.com: 1
  tealiumiq.com: 2
 extended:
  ##.ad-text
  ##.adSense
  ##.adBlock
  ##.adContent
```
</details>

**Comment by mapx-** [comment]
enable the regional list AdGuard Dutch

**Comment by alltimemarr** [comment]
Hi!

This issue is back again. I'm using uBlock Origin Lite.
I had no issues last thursday, but started noticing it today (sunday).

I've always had the AdGuard Dutch list enabled.




These are my settings (I just turned on experimental to check if that would fix it, but normally that's disabled):

name: uBlock Origin Lite
version: 2026.111.1925
browser: Chrome 144 (win)
filtering:
 site: complete
 default: complete
permission: all
rulesets:
 +adguard-mobile
 +adguard-spyware-url
 +block-lan
 +dpollock-0
 +stevenblack-hosts
 +ublock-experimental
 +ubol-tests
 easylist
 easyprivacy
 jpn-1
 nld-0
 pgl
 ublock-badware
 ublock-filters
 urlhaus-full



Rectification: I know for sure it was working on tuesday Jan 13th. I'm having issues on shows like these:
https://www.kijk.nl/programmas/murder-in-a-small-town
https://www.kijk.nl/programmas/cedar-cove

While I can watch a Flemish show without any issues:
https://www.kijk.nl/programmas/flo

So it might just be built into non-dutch productions, i don't know. Last thursday I watched Flo, but today that still works. It's just the american shows that don't.
